package.json

```diff
@@ -14,17 +14,15 @@
         "-p",
         "harfbuzz"
       ]
-    ],
-    "install": [
-      "esy-installer harfbuzz.install"
     ]
   },
   "dependencies": {
-    "@opam/dune": ">=1.6.0",
     "@esy-ocaml/reason": "^3.3.7",
-    "ocaml": ">=4.4.0",
-    "esy-harfbuzz": "^1.9.1005"
+    "@opam/dune": ">=2.0.1",
+    "@opam/dune-configurator": ">=2.0.1",
```

Comment:
Theese constraints allow dune 2.1 and 2.2 to be installed and thoses version don't read dune lang 2.3 so dune-project file should be modified.

Reply:
Resolved in: 6faf802